yt_dlp/extractor/xiaohongshu.py

```diff
@@ -68,6 +69,20 @@ def _real_extract(self, url):
             formats.extend(traverse_obj(info, (('mediaUrl', ('backupUrls', ...)), {
```

Comment:
This should be `masterUrl`, instead; `mediaUrl` doesn't exist afaik (https://github.com/yt-dlp/yt-dlp/issues/9529#issuecomment-2041337087).
```suggestion
            formats.extend(traverse_obj(info, (('masterUrl', ('backupUrls', ...)), {
```